pandas/core/arrays/datetimes.py

```diff
@@ -1628,9 +1628,8 @@ def isocalendar(self) -> DataFrame:
 
         See Also
         --------
-        DatetimeIndex.day: The day of the datetime.
-        DatetimeIndex.minute: The minutes of the datetime.
-        DatetimeIndex.second: The seconds of the datetime.
+        Series.dt.minute : Minutes of the datetime.
+        Series.dt.second : Seconds of the datetime.
```

Comment:
Could you split this PR into 2 separate parts?
I.E. let this PR fix the bug, and make a new PR for improving the docstrings

Reply:
Ok, will do.